scm/doc/TechGuide/chap_quick.rst

```diff
@@ -439,10 +420,31 @@ executed through a Python run script that is pre-staged into the ``bin``
 directory: ``run_scm.py``. It can be used to run one integration or several
 integrations serially, depending on the command line arguments supplied.
 
+Downloading input data
+^^^^^^^^^^^^^^^^^^^^^^
+The various SCM cases require staged input data in order to run. This includes
+input data for cases and lookup tables for runtime use. This is a large dataset
+(:math:`<`\ 1 GB) so it is not stored in the SCM repository, and must be downloaded
+separately. To download this data place it in the correct directories, 
+execute the following scripts:
+
+.. code:: bash
+
+   ./contrib/get_all_static_data.sh
+   ./contrib/get_thompson_tables.sh
+
```

Comment:
With #465 we need to add
 `./contrib/get_aerosol_climo.sh`

Reply:
Should this be a separate script or does it make sense to roll it into the "get_all_static_data" script?

Reply:
This should be separate, IMO, since it is larger than the others and it is not needed unless using GOCART climatology, which is only necessary for some configurations (e.g. latest HR3)